lib/media/segment_index.js

```diff
@@ -308,6 +308,17 @@ shaka.media.SegmentIndex = class {
     this.numEvicted_ += diff;
   }
 
+  /**
+   * Removes all SegmentReferences that end before the given time.
+   * @export
+   */
+  evictAll() {
```

Comment:
If it was exported, it should be defined in `shaka.extern.SegmentIndex`.

Reply:
I removed `evictAll()` in the current implementation as only segmentIndexes which are associated with an active stream are removed.